util/patch/patch_test.go

```diff
@@ -1408,6 +1409,17 @@ func TestPatchHelperForV1beta2Transition(t *testing.T) {
 			t.Log("Patching the object")
 			g.Expect(patcher.Patch(ctx, obj)).To(Succeed())
 
+			// Adding Available as a second condition, but it should be sorted as first
+			t.Log("Creating a new patch helper")
+			patcher, err = NewHelper(obj, env)
+			g.Expect(err).ToNot(HaveOccurred())
+
+			t.Log("Marking clusterv1.conditions and metav1.conditions Available=True")
```

Comment:
Sorry don't want to annoy anyone with nitpicks. Now we log that we set v1beta1 & v1beta2 Available condition to true, but we only actually set v1beta2

Reply:
No worries at all!
I really appreciate all the effort you are putting on reviews